src/quo2/components/record_audio/record_audio/style.cljs

```diff
@@ -221,8 +223,78 @@
    {:margin-bottom 32
     :margin-right  32}))
 
-(def input-container
+(def button-container
   {:width           140
    :height          140
    :align-items     :flex-end
-   :justify-content :flex-end})
\ No newline at end of file
+   :justify-content :flex-end
+   :position        :absolute
+   :right           -10})
+
+(def bar-container
+  {:width  "100%"
+   :height 128})
+
+(defn soundtrack-container
+  []
+  {:height           4
+   :border-radius    2
+   :background-color (colors/theme-colors colors/neutral-20 colors/neutral-80)
+   :overflow         :hidden
+   :position         :absolute
+   :left             80
+   :right            148
+   :bottom           34})
+
+(defn soundtrack-bar
+  [fill-percentage ready-to-delete?]
+  {:width            (str fill-percentage "%")
+   :height           4
+   :border-radius    2
+   :background-color (if ready-to-delete?
+                       (colors/theme-colors colors/danger-50 colors/danger-60)
+                       (colors/theme-colors colors/primary-50 colors/primary-60))})
+
+(defn timer-container
+  [reviewing-audio?]
+  {:position       :absolute
+   :left           (if reviewing-audio? 67 20)
+   :bottom         28.5
+   :flex-direction :row
+   :align-items    :center})
+
+(defn timer-circle
+  []
+  {:width            8
+   :height           8
+   :border-radius    4
+   :margin-right     6
+   :background-color (colors/theme-colors colors/danger-50 colors/danger-60)})
+
+(defn timer-text
+  []
+  {:color (colors/theme-colors colors/danger-50 colors/danger-60)})
+
+(defn player-slider-container
+  []
+  (merge
+   {:position :absolute
+    :left     (if platform/ios? 115 104)
+    :right    (if platform/ios? 108 92)
+    :bottom   (if platform/ios? 16 27)}
+   (when platform/android?
+     ;; Workaround to increase the thickness of the slider track on Android
+     ;; which is currently not supported by the Slider library.
+     {:transform [{:scaleY 2}]})))
```

Comment:
This makes the slider track thicker but on the opposite it stretches the thumb image height to double, so added Android-specific images for the thumb with half the original height and the original width.

Unfortunately the slider component is barely customizable on Android and workarounds are needed to achieve the required design (at least to something close enough).